sormas-cargoserver/build_deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+mvn cargo:stop
```

Comment:
why is the `docker-compose down` not always executed? is it not necessary?

Reply:
no :)